inst/rmd/h/default.html

```diff
@@ -47,13 +47,6 @@
 
 $if(highlightjs)$
 <style type="text/css">code{white-space: pre;}</style>
-$if(theme)$
-<style type="text/css">
-  pre:not([class]) {
-    background-color: white;
-  }
-</style>
-$endif$
```

Comment:
This is duplicated logic (it also appears on line 107). That logic should probably also be pulled out and sass-ified (so it respects the theme's `bg`)